util/fibers/detail/scheduler.cc

```diff
@@ -269,19 +270,22 @@ void Scheduler::Attach(FiberInterface* cntx) {
   fibers_.push_back(*cntx);
 
   if (cntx->type() == FiberInterface::WORKER) {
+    worker_stack_size_ += cntx->stack_size();
     ++num_worker_fibers_;
   }
 }
 
 void Scheduler::DetachWorker(FiberInterface* cntx) {
   fibers_.erase(fibers_.iterator_to(*cntx));
   --num_worker_fibers_;
+  worker_stack_size_ -= cntx->stack_size();
```

Comment:
I assume this is called in the migration path, right? (i.e. that we don't leak)

Reply:
yes